types/src/lib.rs

```diff
@@ -5,6 +5,9 @@
 
 extern crate alloc;
 
+/// Ten megabytes.
+pub const TEN_MB: u32 = 10 * 1024 * 1024;
```

Comment:
I would prefer if we name it `TEN_MB_IN_BYTES` or change documentation to be explicit :)